src/Faker/Provider/en_GB/Person.php

```diff
@@ -90,4 +90,25 @@ class Person extends \Faker\Provider\Person
         'Walker', 'Walsh', 'Ward', 'Watson', 'White', 'Wilkinson', 'Williams', 'Wilson', 'Wood', 'Wright',
         'Young',
     ];
+
+    /**
+     * Generates a random National Insurance number.
+     * @link https://www.gov.uk/hmrc-internal-manuals/national-insurance-manual/nim39110
+     */
+    public function nino(): string
+    {
+        do {
+            $prefixLetters = strtoupper(static::lexify('??'));
+        } while (
+            in_array($prefixLetters[0], ['D', 'F', 'I', 'Q', 'U', 'V'])
```

Comment:
how about choosing a random element from: `['?O', '?D', ?'F', ?'I', '?Q', ?'U', '?V', 'D?', 'F?', 'I?', 'Q?', 'U?', 'V?', 'BG', 'GB', 'NK', 'KN', 'TN', 'NT', 'ZZ']` instead and then using `lexify` on the result?